diff --git a/node_modules/@types/passport/index.d.ts b/node_modules/@types/passport/index.d.ts
index d8bfb6a..a9a83ac 100644
--- a/node_modules/@types/passport/index.d.ts
+++ b/node_modules/@types/passport/index.d.ts
@@ -5,7 +5,12 @@ declare global {
         // eslint-disable-next-line @typescript-eslint/no-empty-interface
         interface AuthInfo {}
         // eslint-disable-next-line @typescript-eslint/no-empty-interface
-        interface User {}
+        interface User {
+            userId : string;
+            role : UserRoles;
+            isActivated : boolean;
+            refreshToken? : string;
+        }
 
         interface Request {
             authInfo?: AuthInfo | undefined;
@@ -79,6 +84,7 @@ declare global {
 }
 
 import express = require("express");
+import { UserRoles } from "src/common/enums/roles.enum";
 
 declare namespace passport {
     type DoneCallback = (err: any, user?: Express.User | false | null) => void;
